app/main.py

```diff
@@ -1,8 +1,18 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
```

Comment:
Using the person's name as a key in the `Person.people` dictionary assumes that all names are unique. If there are duplicate names, this will lead to overwriting previous entries. Consider using a unique identifier instead.